modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase

```diff
@@ -245,6 +245,39 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test SaveImportCustomObjectAsTemplate {
+		task ("Given a custom object is published") {
+			ObjectDefinitionAPI.createAndPublishObjectDefinition(
+				en_US_label = "student",
+				en_US_plural_label = "students",
+				name = "Student",
+				requiredStringFieldName = "name");
+		}
+
+		task ("And Given I create an import task with custom objects") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.importFile(
+				entityType = "Account (v1_0 - Liferay Headless Admin User)",
```

Comment:
The scenario is saying to save the template with a custom object, it this case Student. If it is not possible to do it for now with Student, we can leave the test prepared and @ignore it until the issue is fixed

Reply:
Done changes, thanks!